core/templates/domain/exploration/LostChangeObjectFactory.ts

```diff
@@ -178,6 +189,25 @@ export class LostChange {
     }
     return result;
   }
+
+  getLanguage(): string {
+    let language = '';
+    let supportedLanguages = constants.SUPPORTED_CONTENT_LANGUAGES;
+    if (this.cmd === 'add_written_translation') {
+      for (let i = 0; i < supportedLanguages.length; i++) {
+        if (this.languageCode === supportedLanguages[i].code) {
+          language = supportedLanguages[i].description;
+        }
+      }
+    } else {
+      for (let i = 0; i < supportedLanguages.length; i++) {
+        if (this.newValue === supportedLanguages[i].code) {
+          language = supportedLanguages[i].description;
+        }
+      }
+    }
+    return language;
```

Comment:
Can we user method from existing LanguageUtils service?

Reply:
I tried that but the problem is that type of this newValue as defined in general can be anything else too other than the string, although here we are getting the string only that language utils service function needs only string type values. So I guess it's better to use a separate function. And also then we will need to import languageUtilsService in the lost change dict which is not needed except for this small function here. So I guess we can keep it as it is.